operations/app/terraform/vars/test/variables.tf

```diff
@@ -98,11 +98,11 @@ variable "dns_vars" {
 ##################
 
 variable "app_tier" {
-  default = "PremiumV2"
+  default = "PremiumV3"
```

Comment:
curious what prompted this change?

Reply:
The data ingest team made that change in their "test/prod" environment, so I update the TF to match.